### DISCUSSION TITLE
DNN migration - move a module from a DNN page to a Vanjaro page and...

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1181] [Q&A]
In RC2, if you move a DNN module from a DNN page to a Vanjaro page, it remains intact in the Modules table (it would sometimes be deleted in RC1), but Vanjaro doesn't recognize the new addition to the page for obvious reasons. The question is, will there be a mechanism to manually recognize (import) the "pending module?"

**Comment by meetmandeep** [comment]
> In RC2, if you move a DNN module from a DNN page to a Vanjaro page, it remains intact in the Modules table (it would sometimes be deleted in RC1)

It would eventually be deleted since the module doesn't appear on the page. 

> The question is, will there be a mechanism to manually recognize (import) the "pending module?"

There is no feature to add or move an existing module to Vanjaro Page at this time. This feature was requested before: #371 
You may want to upvote (or add comment) in the original request and we'll consider it in future if there's enough community interest.
